### PR TITLE
Update kr.m3u

### PR DESCRIPTION
**Pull request by Alstruit**
Remove TvHeadend
Convert some links into Youtube channel IDs
Add NaverTv scrapper by @LaneSh4d0w

### CODE REVIEW

**Comment by LaneSh4d0w** [comment]
Naver streams seem to be way more stable than the SeeZN links in so many ways.

**Review by BellezaEmporium — DISMISSED** [review]

**Comment by LaneSh4d0w** [comment]
[연합뉴스TV](https://tv.naver.com/yonhapnewstv) 3214537

**Comment by LaneSh4d0w** [comment]
[채널A뉴스](https://tv.naver.com/channelanews) 3591828

**Comment by Alstruit** [comment]
> [채널A뉴스](https://tv.naver.com/channelanews) 3591828
Not 24/7.

**Review by BellezaEmporium — DISMISSED** [review]

File: streams/kr.m3u [hunk 1]
```diff
@@ -333,3 +333,11 @@ https://kr-streams.onrender.com/scraper?channel=438
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCnfwIKyFYRuqZzzKBDt6JOA/live
 #EXTINF:-1 tvg-id="" status="online",SPOTV (1080p)
 https://bcovlive-a.akamaihd.net/99545824f51f4c1592b35314c6ece3b0/ap-northeast-1/6271486521001/playlist.m3u8
+#EXTINF:-1 tvg-id="ScreenGolfzon.kr" status="online",Screen Golfzon (1080p)
+https://nv-streams.onrender.com/scraper?channel=3077128
+#EXTINF:-1 tvg-id="YTN.kr" status="",YTN (1080p)
```
Comment (LaneSh4d0w) [inline streams/kr.m3u:338]: Not 24/7.

**Review by BellezaEmporium — APPROVED** [review]

**Review by MapGuy11 — APPROVED** [review]

**Pull request merged**